webapp/components/create_comment.jsx

```diff
@@ -576,7 +574,7 @@ export default class CreateComment extends React.Component {
                                 value={this.state.message}
                                 onBlur={this.handleBlur}
                                 createMessage={Utils.localizeMessage('create_comment.addComment', 'Add a comment...')}
-                                emojiEnabled={this.state.emojiPickerEnabled}
+                                emojiEnabled={window.mm_config.EnableEmojiPicker === 'true'}
```

Comment:
@hmhealey I missed your point on your first comment. But I guess it's better to have it in props. Actually for `FileUpload` component, I have a pending PR (https://github.com/mattermost/platform/pull/6837/files) which removed emoji-related items from that component as it's totally unrelated.